win32ss/user/ntuser/defwnd.c

```diff
@@ -863,10 +864,15 @@ IntDefWindowProc(
                   {
                      snapRect.right = (snapRect.left + snapRect.right) / 2;
                   }
-                  else // VK_RIGHT
+                  if (wParam == VK_RIGHT) // VK_RIGHT
                   {
                      snapRect.left = (snapRect.left + snapRect.right) / 2;
                   }
+				  if (wParam == VK_F2)
+                  {
+                     snapRect.right = (snapRect.left + snapRect.right) / 2;
+					 snapRect.bottom = (snapRect.top + snapRect.bottom) / 2;
```

Comment:
```suggestion
                     snapRect.bottom = (snapRect.top + snapRect.bottom) / 2;
```
(fixed tabs vs. spaces)